src/analysis/inner_product_space/basic.lean

```diff
@@ -432,7 +431,9 @@ lemma real_inner_smul_right {x y : F} {r : ℝ} : ⟪x, r • y⟫_ℝ = r * ⟪
 lemma inner_smul_real_right {x y : E} {r : ℝ} : ⟪x, (r : 𝕜) • y⟫ = r • ⟪x, y⟫ :=
 by { rw [inner_smul_right, algebra.smul_def], refl }
 
-/-- The inner product as a sesquilinear form. -/
+/-- The inner product as a sesquilinear form.
+
+Note that in the case `𝕜 = ℝ` this is a bilinear form. -/
```

Comment:
Can you include the recipe to obtain a `bilin_form` from this?

Reply:
Right now `bilin_form` is quite heavily linked to `quadratic_form`, so we can't burn all the bridges to `bilin_form` just yet.

Reply:
I would not be against having this PR open for a bit until I have removed the dependency of `bilin_form` in `quadratic_form`. I was looking at that as well, but I thought this one was super easy to do.

Reply:
I've added a dependence, so that it does not sit in the review queue